### ISSUE TITLE
Outdated example

### ISSUE THREAD

**Issue by dschnare** [buildkite/nodejs-docker-example#1]
Isn't the `pipeline.yml` file outdated? Shouldn't it be:

```yml
steps:
  - name: ":node:"
    command: "npm test"
    plugins:
      docker-compose#v2.1.0:
        run: app
```

**Comment by dschnare** [comment]
See: https://buildkite.com/docs/builds/docker-containerized-builds

**Comment by toolmantim** [comment]
Thanks @dschnare! Fixed in cfc7fafb3c08259ff1a72dbbad3c2c56b0d15ca1
